src/cose_int.h

```diff
@@ -159,91 +167,227 @@ extern cose_error _MapFromCBOR(cn_cbor_errback err);
+                              const byte* pbKeyIn,
+                              size_t cbKeyIn,
+                              const char* szContext,
+                              cose_errback* perr);
+extern bool _COSE_Mac_validate(COSE_MacMessage* pcose,
+                               COSE_RecipientInfo* pRecip,
+                               const byte* pbKeyIn,
+                               size_t cbKeyIn,
+                               const char* szContext,
+                               cose_errback* perr);
 
 //  MAC0 Items
-extern HCOSE_MAC0 _COSE_Mac0_Init_From_Object(cn_cbor *, COSE_Mac0Message * pIn, CBOR_CONTEXT_COMMA cose_errback * errp);
-extern bool _COSE_Mac0_Release(COSE_Mac0Message * p);
+extern HCOSE_MAC0 _COSE_Mac0_Init_From_Object(
+    cn_cbor*,
+    COSE_Mac0Message* pIn,
+    CBOR_CONTEXT_COMMA cose_errback* errp);
+extern bool _COSE_Mac0_Release(COSE_Mac0Message* p);
 
 //  Counter Sign Items
-extern HCOSE_COUNTERSIGN _COSE_CounterSign_get(COSE * pMessage, int iSigner, cose_errback * perr);
-extern bool _COSE_CounterSign_add(COSE * pMessage, HCOSE_COUNTERSIGN hSigner, cose_errback * perr);
-extern bool _COSE_CountSign_create(COSE * pMessage, cn_cbor * pcnBody, CBOR_CONTEXT_COMMA cose_errback * perr);
+extern HCOSE_COUNTERSIGN _COSE_CounterSign_get(COSE* pMessage,
+                                               int iSigner,
+                                               cose_errback* perr);
+extern bool _COSE_CounterSign_add(COSE* pMessage,
+                                  HCOSE_COUNTERSIGN hSigner,
+                                  cose_errback* perr);
+extern bool _COSE_CountSign_create(COSE* pMessage,
+                                   cn_cbor* pcnBody,
+                                   CBOR_CONTEXT_COMMA cose_errback* perr);
 
 //
 //  Debugging Items
 
 //#define DO_ASSERT assert(false);
 #define DO_ASSERT
-#define CHECK_CONDITION(condition, error) { if (!(condition)) { DO_ASSERT; if (perr != NULL) {perr->err = error;} goto errorReturn;}}
-#define CHECK_CONDITION0(condition, error) { if ((condition)) { DO_ASSERT; if (perr != NULL) {perr->err = error;} goto errorReturn;}}
-#define FAIL_CONDITION(error) { DO_ASSERT; if (perr != NULL) {perr->err = error;} goto errorReturn;}
-#define CHECK_CONDITION_CBOR(condition, error) { if (!(condition)) { DO_ASSERT; if (perr != NULL) {perr->err = _MapFromCBOR(error);} goto errorReturn;}}
-
-extern cn_cbor * _COSE_encode_protected(COSE * pMessage, cose_errback * perr);
-
+#define CHECK_CONDITION(condition, error) \
+  {                                       \
+    if (!(condition))                     \
+      {                                   \
+        DO_ASSERT;                        \
+        if (perr != NULL)                 \
+          {                               \
+            perr->err = error;            \
+          }                               \
+        goto errorReturn;                 \
+      }                                   \
+  }
```

Comment:
style is very subjective, though I think here the majority of people agree that this is more readable